apps/omg_performance/lib/omg_performance/http_rpc/watcher_client.ex

```diff
@@ -14,72 +14,167 @@
 
 defmodule OMG.Performance.HttpRPC.WatcherClient do
   @moduledoc """
-  Provides access to Watcher's RPC API
+  Provides access to Watcher's RPC API, as required by the `OMG.Performance` tool
+
+
+  TODO: This module includes a big fat copy paster from `Support.WatcherHelper` - do sth about this later
```

Comment:
no need to couple them, have a separate implementation if the requirements are different.

Reply:
I'll change `TODO` to `NOTE` then, if >1 person is comfortable with this :)